ghzdac/correction.py

```diff
@@ -1110,6 +1130,9 @@ def DACifyFT(self, signal, t0=0, n=8192, offset=0, nfft=None, loop=False,
         decayRates = np.array([x['GHz'] for x in self.decayRates])
         decayAmplitudes = self.decayAmplitudes
 
+        reflectionRates = np.array([x['GHz'] for x in self.reflectionRates])
```

Comment:
Yeah, the deconv server doesn't do units, so I guess this is how units were dealt with. I found it like this for the settling part, copied that for the reflection part.